src/stepDefinitions/filesContext.js

```diff
@@ -270,3 +271,23 @@ Given(/^user "([^"]*)" has created folder "(.+)"$/, function(userId, folderName)
   folderName = folderName.replaceAll("\\", "")
   return webdavHelper.createFolder(userId, folderName)
 })
+
+Then(
+  'as user {string} folder {string} should contain {string} items', async function (
+    userName,
+    folderName,
+    itemsCount
+) {
+  const davPath = webdavHelper.createDavPath(userName, folderName)
+  const resourcesInsideFolder = await webdavHelper.propfind(davPath, userName, [])
+  let jsonResponse = convert.xml2json(resourcesInsideFolder)
+  jsonResponse = JSON.parse(jsonResponse)
+  let actualCount = jsonResponse.elements[0].elements.length
+  actualCount = actualCount - 1
```

Comment:
`xml-js` comes with `xml2js` function so you can directly convert XML to js object.
```suggestion
  const xmlResponse = await webdavHelper.propfind(davPath, userName, [])
  const resObj = convert.xml2js(xmlResponse)
  // also add some comment here why we need to subtract by 1
  const actualCount = resObj.elements[0].elements.length - 1
```